### PR TITLE
Items and blocks mapped.

### PR DESCRIPTION
**Pull request by mineLdiver**
From @KydZombie
```
All of the item and block instances have been named as well as the majority of their classes. Lots of small things that I probably can't even list.
```

### CODE REVIEW

File: mappings/net/minecraft/block/BlockSoundGroup.mapping [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_475 net/minecraft/block/BlockSoundGroup
```
Comment (mineLdiver) [inline mappings/net/minecraft/block/BlockSoundGroup.mapping]: Supposed to be `net/minecraft/sound/BlockSoundGroup`, according to https://github.com/FabricMC/yarn/blob/23w35a/mappings/net/minecraft/sound/BlockSoundGroup.mapping

File: mappings/net/minecraft/Dimension.mapping [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_50 net/minecraft/Dimension
```
Comment (mineLdiver) [inline mappings/net/minecraft/Dimension.mapping]: No package

File: mappings/net/minecraft/ToolMaterial.mapping [hunk 3]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_428 net/minecraft/ToolMaterial
```
Comment (mineLdiver) [inline mappings/net/minecraft/ToolMaterial.mapping]: No package

File: mappings/net/minecraft/NetherDimension.mapping [hunk 4]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_502 net/minecraft/NetherDimension
```
Comment (mineLdiver) [inline mappings/net/minecraft/NetherDimension.mapping]: No package

File: mappings/net/minecraft/Translator.mapping [hunk 5]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_629 net/minecraft/Translator
```
Comment (mineLdiver) [inline mappings/net/minecraft/Translator.mapping]: No package. Also, `I18n` seems to fit yarn better.

**Review by mineLdiver — COMMENTED** [review]
Missing packages. Some names can be closer to yarn.

**Review by calmilamsy — APPROVED** [review]
Looks good.

**Pull request merged**